src/components/modals/ReconfigureFCModal.tsx

```diff
@@ -131,10 +173,31 @@ export default function ReconfigureFCModal({
     ticketingForm.setFieldsValue({
       reserved: parseFloat(fromPerbicent(metadata.reservedRate)),
     })
-  }, [dispatch, fundingCycle, payoutMods, ticketMods, ticketingForm])
+
+    if (metadata.version === 1) {
```

Comment:
**question**: version in metadata is `int` (`1`), but previously I noticed it was a string (`'1'`). Is this correct?

Reply:
FundingCycleMetadata version is an `int` (maybe it used to be a string?) but you may be thinking of the Terminal contract version, which is a string `"1" | "1.1"`. My reasoning for those is that the FC metadata version is stored in the contracts as a number, whereas the terminal version is more like a name. Although we could treat 1 and 1.1 as numbers, there's nothing to prevent us from launching a `1.1.1`, or a `1.1-<string>`